proto/v1/database_service.proto

```diff
@@ -949,6 +949,10 @@ message TableConfig {
   // Format: users/{email}
   string updater = 4 [(google.api.field_behavior) = OUTPUT_ONLY];
 
+  // The last change come from branch.
+  // Format: branches/{branch}
```

Comment:
Either projects/{project}/branches/{branch} or {branch-id}

{branch-id} is preferred. Save some bytes.

Reply:
Use projects/{project}/branches/{branch} to keep consistent with plan.